DrivewayDentDeletion/Operators/readme.md

```diff
@@ -24,14 +21,13 @@ These steps will need to be documented in the demo docs:
 - Apply yaml to create the pipeline, configured to use the forked repo. Set
 `FORKED_REPO` to the URL for your repo.
   ```
-  oc project driveway-dent-deletion
+  oc project <NAMESPACE>
```

Comment:
Need to export NAMESPACE so it can be used in the sed below

Reply:
Done